invenio_app_ils/templates/invenio_app_ils/backoffice.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    {% assets "invenio_app_ils_backoffice_css" %}
+    <link href="{{ ASSET_URL }}" rel="stylesheet">{% endassets %}
+</head>
+
+<body>
+    <div id="backoffice"></div>
+    {{ jwt() }}
+</body>
+<!-- react-scripts > 2 needs the following script to load the different bundle chunks correctly -->
```

Comment:
If there is an un minified version of these react-scripts would be nice.

Reply:
Unfortunately didn't find something but we could create a shared template to include this script once. I will update the code.